static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -1,69 +1,119 @@
 define([
         'lib/config',
         'lib/storage',
-        'lodash/objects/assign'
-    ], function(config,
-                storage,
-                assign) {
+        'lodash/objects/assign',
+        'lib/geolocation'
+    ], function(
+        config,
+        storage,
+        assign,
+        geolocation
+    ) {
 
-    function selectSequentiallyFrom(array) {
-        return array[storage.local.get('gu.alreadyVisited') % array.length];
+    var offerings = {
+        membership: 'membership',
+        contributions: 'contributions'
+    };
+
+    function defaultColourStrategy() {
```

Comment:
Ah I see it why it ended up like this - I guess I would just put it straight on `baseParams` now